PROJECT_PROGRESSION.md

```diff
@@ -16,14 +16,68 @@ This governance policy sets forth the proposal process for projects to be accept
 
-#### Application Phases
+#### Roles
+
+##### Applicant
+
+Individual or group who submits the application on behalf of a project to join the OpenJS Foundation.
+
+##### Application Manager
+
+The application manager ensures progress through the process, helps the project understand the work to be completed and
+to move though the application process. Typically the application manager will be one of the Foundation staff members.
+
+##### Application Champion
+
+The application champion is a member of the CPC who commits to do the following:
+
+1. Provide weekly update to the applicant on progression through the process.
+1. Be available to answer questions from the applicant.
+1. Advocate on behalf of the project applying.
+1. Help to explain why the project is a good fit.
+1. Help the application manager when needed.
+
+
+#### Process
 
 1. Initial email sent to new-projects@lists.openjsf.org with filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md).
+1. An acknowledgement is sent to the applicant by Foundation staff within 48 hours (striving to reply within 24 hours when possible).
 1. Silent period. Internal only discussion.
-1. Initial acceptance as [Incubating][] Project.
+1. Initial acceptance as an [Incubating][] Project. Before public announcement there will be:
+   * Private email to the CPC members which asks for +1 or objections to the application becoming public. This email will include:
+     * The name of the application manager for the application
+     * The name of the application champion for the application
+     * Completed application 
+     * Completed application-fit checklist.
+     * Confirmation that the board has had an FYI and there were no objections.
+     * Confirmation that there are no existing objections from CPC members.
+     This is not a formal vote just the process to ensure we have regular CPC concensus to move forward.
+   * If there is agreement to move forward:
+     * Confirmation to the applicant that they are ready to have their application become public.
+     * Discussion/agreement between applicant, application manager and Foundation media team to agree on announce date.
+       Confirm next steps beyond announcement.  
+   * If lack of consensus to move forward:
+      * Correspondence from foundation staff to applicant about non-acceptance.
+      * If possible, give guidance to the project in their quest to find a foundation home
+        (ex. perhaps a different foundation would be a more suitable endeavor.)
+   * Discussion/agreement between applicant, application manager and Foundation media team to agree on announce date.
 1. Project goes through process of adhering to [on-boarding checklist](#onboarding-checklist).
 1. Project presents to CPC for final acceptance via 2/3 supermajority vote and appropriate initial stage is determined.
 
+##### Initial fit checklist
+
```

Comment:
We should likely include something below about initial review of items to be legally transferred

e.g. 

* What Trade Marks will be transferred
* What domains will be transferred
* What IP currently exists and how was it licensed. ( this one is similar to existing point about license being compatible)

Reply:
@MylesBorins , do those 3 need to be captured/resolved during the quiet period?  If so we should add to the list, if not then maybe better in a checklist for the final vote?

Reply:
I think this is something to check with board / legal. We likely want to do a very high level review to ensure their are no obvious conflicts

Reply:
@MylesBorins added those three, and we can ask board about legal review in the next meeting

Reply:
We discussed in board meeting, I think the answer was no.  @MylesBorins FYI.

Reply:
We discussed in the recent board meeting and I think the answer was that we don't need a high-level legal review during quiet period.  Any required review will be handled during the incubation phase.